modules/standard/IO.chpl

```diff
@@ -3484,31 +3458,26 @@ proc stringify(const args ...?k):string {
     return str;
   } else {
     // otherwise, write it using the I/O system.
-
     try! {
       // Open a memory buffer to store the result
       var f = openmem();
+      defer try! f.close();
```

Comment:
These 3 `defer try! close()` statements are already inside of a `try!` block.  Do they really need another `try!`?  Is that because they are deferred?

Reply:
It's because they're deferred, yeah. The rationale for this requirement is in #8161.